internal/controlplane/handlers_repositories_test.go

```diff
@@ -37,6 +37,7 @@ import (
 	mockmanager "github.com/stacklok/minder/internal/providers/manager/mock"
 	ghrepo "github.com/stacklok/minder/internal/repositories/github"
 	mockghrepo "github.com/stacklok/minder/internal/repositories/github/mock"
+	rf "github.com/stacklok/minder/internal/repositories/github/mock/fixtures"
```

Comment:
The diff for this file should just be refactoring, I moved helper routines to this package.